src/sagemaker/estimator.py

```diff
@@ -347,6 +347,7 @@ def deploy(self, initial_instance_count, instance_type, accelerator_type=None, e
             endpoint_name (str): Name to use for creating an Amazon SageMaker endpoint. If not specified, the name of
                 the training job is used.
             use_compiled_model (bool): Flag to select whether to use compiled (optimized) model. Default: False.
+            update_endpoint (bool): Weather to update an endpoint or create a new endpoint. Default: False.
```

Comment:
'Flag to update the model in an existing Amazon SageMaker endpoint. If True, this will deploy a new EndpointConfig to an already existing endpoint and delete resources corresponding to the previous EndpointConfig. Default: False'

Reply:
https://docs.aws.amazon.com/sagemaker/latest/dg/API_UpdateEndpoint.html